src/snowflake/connector/network.py

```diff
@@ -247,7 +248,7 @@ def raise_failed_request_error(
 
 
 def is_login_request(url: str) -> bool:
-    return "/session/v1/login-request" in url
+    return "login-request" in parse_url(url).path
```

Comment:
may I ask why this is changed?

Reply:
[I just used Harsh's implementation.](https://github.com/snowflakedb/snowflake-connector-python/blob/4777d83f8cac79194f890b34e50162e35478ee00/src/snowflake/connector/network.py#L1162) I though it looked a little more robust.